spark3/src/test/java/org/apache/iceberg/actions/TestCreateActions.java

```diff
@@ -70,6 +79,10 @@
 
   private static final String NAMESPACE = "default";
 
+  // Error message
+  private static final String OUTPUT_DIFFERENT_ERROR_MSG =
```

Comment:
nit: I think it is alright to use the error message in place. To stay on one line, we can simplify the message like `Output must match` or anything appropriate.